azure-mgmt-batchai/src/main/java/com/microsoft/azure/management/batchai/BatchAIJob.java

```diff
@@ -269,7 +269,13 @@
         /**
          * The first stage of Batch AI job definition.
          */
-        interface Blank extends DefinitionWithRegion<WithNodeCount> {
+        interface Blank extends WithCluster {
+        }
+
+        interface WithCluster {
+            WithNodeCount withBatchAICluster(BatchAICluster cluster);
```

Comment:
we usually use the prefix "Existing" when referring an existing resource `withExistingBatchAICluster(..)`